File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoLayoutLocmapLineStroke.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnThemeStroke;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoLayoutLocmapLineStroke
{
  @Nullable
  public DefnDtoColor color;

  @Nullable
  public MetaIdField colorFieldId;

  @Nullable
  public DefnDtoColor colorVar;

  @Nullable
  public MetaIdField groupByFieldId;

  @Nullable
  public EnumDefnThemeStroke stroke;

  @Nullable
  public MetaIdField strokeFieldId;

  @Nullable
  public EnumDefnThemeStroke strokeVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoLayoutLocmapLineStroke.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoColor
import com.neome.api.meta.base.Types.EnumDefnThemeStroke
import com.neome.api.meta.base.Types.MetaIdField

interface DefnDtoLayoutLocmapLineStroke
{
  val color: DefnDtoColor?
  val colorFieldId: MetaIdField?
  val colorVar: DefnDtoColor?
  val groupByFieldId: MetaIdField?
  val stroke: EnumDefnThemeStroke?
  val strokeFieldId: MetaIdField?
  val strokeVar: EnumDefnThemeStroke?
}